src/actions/config.ts

```diff
@@ -0,0 +1,18 @@
+import {VoyagerConfig} from '../models/config';
+import {ReduxAction} from './redux-action';
+
+export type ConfigAction = SetConfig;
+
+export const SET_CONFIG = 'SET_CONFIG';
+export type SetConfig = ReduxAction<typeof SET_CONFIG, {
+  config: VoyagerConfig
+}>;
+
+export function setConfig(config: VoyagerConfig): SetConfig {
```

Comment:
FYI, since there are already interface for actions like `SetConfig`, I normally don't create action creator for synchronous actions.